src/klein/test/py3_test_resource.py

```diff
@@ -1,12 +1,12 @@
 import twisted
+from twisted.trial.unittest import TestCase as AsynchronousTestCase
 
 from .test_resource import LeafResource, _render, requestMock
-from .util import TestCase
 from .. import Klein
 from .._resource import KleinResource
 
 
-class PY3KleinResourceTests(TestCase):
+class PY3KleinResourceTests(AsynchronousTestCase):
```

Comment:
Separately from what's going on in this branch, we should really make this test synchronous (it doesn't need to be async).